src/equations/visco_resistive_mhd_3d.jl

```diff
@@ -0,0 +1,210 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+@doc raw"""
+    ViscoResistiveMhd3D(gamma, inv_gamma_minus_one,
+                               μ, Pr, eta, kappa,
+                               equations, gradient_variables)
+
+These equations contain the viscous Navier-Stokes equations coupled to
+the magnetic field together with the magnetic diffusion applied
+to mass, momenta, magnetic field and total energy together with the advective terms from
+the [`IdealGlmMhdEquations3D`](@ref).
+Together they constitute the compressible, viscous and resistive MHD equations with energy.
+
+- `gamma`: adiabatic constant,
+- `mu`: dynamic viscosity,
+- `Pr`: Prandtl number,
+- `eta`: magnetic diffusion (resistivity)
+- `equations`: instance of the [`IdealGlmMhdEquations3D`](@ref)
+- `gradient_variables`: which variables the gradients are taken with respect to.
+Defaults to `GradientVariablesPrimitive()`.
+
+Fluid properties such as the dynamic viscosity $\mu$ and magnetic diffusion $\eta$
+can be provided in any consistent unit system, e.g.,
+[$\mu$] = kg m⁻¹ s⁻¹.
+
+The equations given here are the visco-resistive part of the MHD equations
+in conservation form:
+```math
+\overleftrightarrow{f}^\mathrm{\mu\eta} =
+\begin{pmatrix}
+\overrightarrow{0} \\
+\underline{\tau} \\
+\underline{\tau}\overrightarrow{v} - \overrightarrow{\nabla}q -
+ \eta\left( (\overrightarrow{\nabla}\times\overrightarrow{B})\times\overrightarrow{B} \right) \\
+\eta \left( (\overrightarrow{\nabla}\overrightarrow{B})^\mathrm{T} - \overrightarrow{\nabla}\overrightarrow{B} \right) \\
+\overrightarrow{0}
+\end{pmatrix},
+```
```

Comment:
This looks fine, but I realized after my previous comment that the ideal GLM-MHD equations do not contain proper docstrings for the equation formulation and their description. Therefore, I think we can save a "proper" docstring writing for the ideal GLM-MHD equations as well as the new visco-resistive GLM-MHD equations for a different PR. This way we can be sure that the notation across all files is consistent and easy to review. What do you think @sloede @ranocha ?

Reply:
Sounds good. Maybe we can keep this one and change it in case we need to.

Reply:
:+1: